packages/react-components/react-motion/library/src/hooks/useAnimateAtoms.ts

```diff
@@ -14,13 +18,19 @@ function useAnimateAtomsInSupportedEnvironment() {
       const { isReducedMotion } = options;
 
       const animations = atoms.map(motion => {
-        const { keyframes, ...params } = motion;
-        const animation = element.animate(keyframes, {
-          fill: 'forwards',
+        const { keyframes, reducedMotion, ...params } = motion;
+        const { keyframes: reducedMotionKeyframes = keyframes, ...reducedMotionParams } = reducedMotion ?? {};
 
+        const animationKeyframes: Keyframe[] = isReducedMotion ? reducedMotionKeyframes : keyframes;
+        const animationParams: KeyframeEffectOptions = {
+          ...DEFAULT_ANIMATION_OPTIONS,
           ...params,
+
           ...(isReducedMotion && { duration: 1 }),
-        });
+          ...(isReducedMotion && reducedMotionParams),
```

Comment:
If a user has provided reducedMotion params, I think we should stop applying the default `duration: 1` value.

It might sound stupid to pass no duration and it defaults to 0, but we should still the user have the full override power in this case

Reply:
> If a user has provided reducedMotion params, I think we should stop applying the default duration: 1 value.

I wanted to avoid breaks, that's why I kept it. A user will be still able to override `duration`, so I don't see any harm with the current behavior.

Reply:
@ling1726 @layershifter What if we forced them to override `duration` for `reducedMotion`? After all, if the duration stays at 1 ms, new keyframes aren't going to accomplish much, in most use cases. And there might be someone who adds `reducedMotion` keyframes, then is puzzled when they don't seem to work, because they don't realize the duration is at 1 ms and they need to change it.

Reply:
> @ling1726 @layershifter What if we forced them to override `duration` for `reducedMotion`? After all, if the duration stays at 1 ms, new keyframes aren't going to accomplish much, in most use cases. And there might be someone who adds `reducedMotion` keyframes, then is puzzled when they don't seem to work, because they don't realize the duration is at 1 ms and they need to change it.

That's valid concern. As both you (@robertpenner) and @ling1726 raised it, I updated the implementation:
- `reducedMotion` is `undefined` => `{ duration: 1ms }`
- `reducedMotion` is defined => we will use whatever user provided